### PR TITLE
RHA/QRW-71/profile_page_backend

### PR DESCRIPTION
**Pull request by RustamHabibovv**
All the subtasks are performed here are some notes regarding work:
1.) I did not add any profile button to home page, as I am responsible for home page, so therefore I did not want to interrupt others'work, so for now you have to manually go /profie url to acces profile page
2.) As I just finished backend, htmls implemented are very simple and maybe a little bit messy, they are just standing for to visualize features implemented
3.)Two new database model created Profile and Page Profile has one to one user relationship Page has many to one relationship when new user is created automatically new profile page is created and assigned, which is handled by signals.py.
4.) In profile page you can change password , change profile image, delete profile image, create new page and see pages you already have and delete which you want. When user delete his profile image, default profile image is automatically assigned.
5.) I know that it is not good that when user clicks change password or change image it is directed to form in new url, as I need to implement some js code to html files, I will do it in frontend task, so in final form every change will happen in profile page, forms will appear here.

### CODE REVIEW

**Review by Strexas — CHANGES_REQUESTED** [review]
Remove __pycache__ from PR and make pylint, mypy check succesful

File: qr_web/urls.py [hunk 1]
```diff
@@ -14,11 +14,21 @@
...
+urlpatterns: URLList = [
     path('admin/', admin.site.urls),
     path('', include('entry.urls')),
     path('social-auth/', include('social_django.urls', namespace='social')),
+    path('profile/', include('user_profile.urls')),
+
 ]
+
+if settings.DEBUG:
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
[the remark this answers is not in the document]
Comment (RustamHabibovv) [reply qr_web/urls.py:34]: no it will not work.

File: qr_web/urls.py [hunk 2]
```diff
@@ -14,11 +14,21 @@
...
-from django.contrib import admin
-from django.urls import path, include
+import typing
 
-urlpatterns = [
+from django.contrib import admin
+from django.urls import path, include,URLResolver,URLPattern
+from django.conf.urls.static import static
+from django.conf import  settings
+Url = typing.Union[URLPattern, URLResolver]
```
[the remark this answers is not in the document]
Comment (RustamHabibovv) [reply qr_web/urls.py:23]: can you elaborate more, I did not get exactly what you mean?

[the remark this answers is not in the document]
Comment (Strexas) [reply qr_web/urls.py:24]: `from typing import Union`

[the remark this answers is not in the document]
Comment (Strexas) [reply qr_web/urls.py:24]: `Url = Union[URLPattern, URLResolver]`

**Review by Strexas — APPROVED** [review]

**Pull request merged**